### ISSUE TITLE
Открыть редактирование автора

### ISSUE THREAD

**Issue by ekorolev** [ekorolev/pesennik#5]
На странице редактирования песни открыть поле автора для редактирования и продумать как это будет обрабатываться на сервере.

**Comment by alekskorolev** [comment]
Помоему пофикшено.
В догонку, когда редактируем песню форматирование теряется.
пример потерянного форматирования
http://pesennik.korolev.life/#/song/56943fbb38c8a8f94e360093
И может быть добавить в футер ссылку на багтрекер здесь? вдруг из пользователей кто не знает куда про баги писать? )

**Comment by ekorolev** [comment]
Нет, не пофикшено. В формочке изменять можно, а вот на сервере данные просто не используютс об имени автора при апдейте песни

С форматированием надо разобраться в чем баг.
